packages/wire-format/src/types.ts

```diff
@@ -142,8 +142,9 @@ export const isVirtuallyFund = guard<VirtuallyFund>('VirtuallyFund');
 export const isFundGuarantor = guard<FundGuarantor>('FundGuarantor');
 
 // channel requests
-type GetChannel = {type: 'GetChannel'; channelId: Bytes32};
-export type ChannelRequest = GetChannel;
+type GetChannel = {type: 'GetChannel'; channelId: string};
+type ProposeLedger = {type: 'ProposeLedger'; nonce: number; channelId: string; outcome: Outcome};
```

Comment:
Shouldn't this be `ProposeLedgerUpdate`? You're not proposing a _ledger_, you're proposing the next outcome in an existing ledger.